build-manual/index.html

```diff
@@ -42,7 +42,7 @@
 <div class="title">Intel&reg; XED Build Guide </div>  </div>
 </div><!--header-->
 <div class="contents">
-<div class="textblock"><p>April 2024</p>
+<div class="textblock"><p>September 2024</p>
```

Comment:
will submit a PR for this change to reflect last time it was edited (April->September)